src/backend/ml_api/README.md

```diff
@@ -2,25 +2,32 @@
 Create env for prompt_api:
 
 ```
-conda create -n cruise_prompts python=3.8.15
+conda create -n cruise_ml_api python=3.8.15
```

Comment:
Since you're starting from scratch, feel free to update Python to version 3.11 or 3.12 at any point

Reply:
Okay, we will upgrade it in the next PR, when we will be implementing more models to the APP